src/connector.rs

```diff
@@ -221,28 +221,28 @@ mod tests {
 
     #[tokio::test]
     async fn connects_https() {
-        oneshot(https_or_http_connector(), https_uri())
+        oneshot(https_or_http_connector(), true)
```

Comment:
Maybe a two variant enum instead of a `bool`? I think the previous version with the helper made it more obvious which invocation was being handed a `https://` uri vs an `http://` uri.

Reply:
I considered it, but it feels a little like overkill? This is not library API but just a test function with the tests right next to it? I doubt these functions will see much change...

Reply:
I don't see the harm in being more explicit, especially in an area where there was a previous oversight. It's a minor nit though, so whatever you want is fine with me.

Reply:
Never mind, I think the enums are worth it.